src/main/java/by/exadel/internship/entity/Internship.java

```diff
@@ -19,8 +21,11 @@
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
+@SQLDelete(sql = "UPDATE internship SET inship_deleted=true WHERE inship_id=?")
+@Where(clause = "inship_deleted = false")
 public class Internship {
 
+    //Why we don't use a GeneratedValue
```

Comment:
Good, you can ask the person who commited this code

Reply:
The person answered that he fixed it